### PR TITLE
Adding Tiktok

### PR DESCRIPTION
**Pull request by SeraphicRav**
#### Description

- Closes #2255 
- What does this PR change? Adding TikTok in social links 
- Did you change something visual? A before/after screenshot can be helpful.
image
Just edited locally `docs/astro.config.mjs`.

### CODE REVIEW

**Review by at-the-vr — COMMENTED** [review]
LGTM 👍 One thing, I am unsure of [optimizing](https://jakearchibald.github.io/svgomg/) this one and will leave that to fellow maintainers

File: .changeset/rude-pets-complain.md
Comment (HiDeoo) [inline .changeset/rude-pets-complain.md:2]:
```suggestion
'@astrojs/starlight': patch
```

We usually release social icons in patches.

File: .changeset/rude-pets-complain.md
Comment (HiDeoo) [inline .changeset/rude-pets-complain.md:5]:
```suggestion
Adds TikTok social icon
```

We always use present tense verbs in changeset for consistency (if interested, our [changeset conventions](https://contribute.docs.astro.build/docs-for-code-changes/pull-requests/#changesets) can be found here).

**Review by HiDeoo — APPROVED** [review]
Thanks for you contribution :raised_hands:

I left and applied tiny suggestions to the changeset, mostly to match our [changeset conventions](https://contribute.docs.astro.build/docs-for-code-changes/pull-requests/#changesets) so that the PR is ready to be merged as it looks perfect to me.

Note that this PR will only be merged when the next Starlight version is released so that the documentation is only updated when the feature is available to users.

> One thing, I am unsure of [optimizing](https://jakearchibald.github.io/svgomg/) this one and will leave that to fellow maintainers

Just ran the SVG in SVGOMG and it results to the same output so we should be good on that front. Thanks for the review too :thumbsup:

**Comment by SeraphicRav** [comment]
Ok, thank you very much !

**Pull request merged**
